pypose/module/mpc.py

```diff
@@ -0,0 +1,49 @@
+import torch
+import pypose as pp
+import torch.nn as nn
+
+
+class MPC(nn.Module):
+
+    def __init__(self, system, T, step=10, eps=1e-4):
+        super().__init__()
+        self.system = system
+        self.T = T
+        self.step = step
+        self.eps = eps
+
+    def forward(self, Q, p, x_init, current_x, current_u, time):
```

Comment:
Like I said, do not put current_x, current_u here. You should generate the initial sequence inside the class rather than let the user generate the non-sense sequences by themselves. See the original mpc.pytorch code.

Also, in the original code, the author always only use the u sequence from last iteration to re-generate x sequence. Should we also do that or we keep using the x sequence from last iteration? Let's double check the math.